src/modules/storage/LedgerStorage.ts

```diff
@@ -2412,26 +2413,23 @@ export class LedgerStorage extends Storages {
      * of the returned Promise is called with the records
      * and if an error occurs the `.catch` is called with an error.
      */
-    public getValidatorsAPI(
-        height: Height | null,
-        address: string | null,
-        conn: mysql.PoolConnection | undefined = undefined
-    ): Promise<any[]> {
+    public getValidatorsAPI(height: Height | null, address: string | null, conn: mysql.PoolConnection | undefined = undefined, limit?: number, page?: number): Promise<any[]> {
         let cur_height: string;
 
+        const LIMIT_QUERY = limit ? page ? `LIMIT ${limit} OFFSET ${limit * (page - 1)}` : '' : '';
         if (height !== null) cur_height = height.toString();
         else cur_height = `(SELECT MAX(height) as height FROM blocks)`;
 
         let sql =
             `SELECT V.address,
                 V.enrolled_at,
-                V.address,
                 V.utxo_key,
                 V.utxo_key as stake,
                 V.amount as stake_amount,
                 P.block_height,
                 P.block_height as preimage_height,
                 P.preimage_hash,
+                V.slashed,
```

Comment:
It was added from the previous commit.

Reply:
Validator slashing ==> this is for frontend on nodes page